LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -13,12 +13,17 @@ public class SpinnerController : Controller
         //TODO: 0) Register the Repository class as a service in Program.cs 
         //TODO: 1) add an instance variable here of type Repository
 
+        Repository repository = new Repository();
 
         /***
          * Constructor - TODO: 2) call for a DIJ Repository object to be passed to the constructor
          **/
-        public SpinnerController()
+
+
+
+        public SpinnerController(Repository repository)
```

Comment:
Good work here - you receive the DIJ object and assign it to your instance variable